### PR TITLE
Feature/prepare for deployment

### PR DESCRIPTION
**Pull request by DominikGuzei**
Please merge https://github.com/meteor-space/donations/pull/32 first.

This PR includes some final preparations for the deployment on scalingo:
- Fixes problems with tap:i18n
- Moves images into `public` folder so that they can be referenced correctly
- includes `force-ssl` for http -> https redirects

https://www.sachspenden.online/

### CODE REVIEW

File: packages/app/source/shared/startup.js [hunk 1]
```diff
@@ -3,6 +3,11 @@ Meteor.startup(function() {
   Donations.app.start();
   FlowRouter.initialize();
 
+  if (Meteor.isServer) {
+    // Configure force-ssl redirect URL
```
Comment (rhyslbw) [inline packages/app/source/shared/startup.js:7]: Cool, this will be easy enough to refector for generic installations later

Comment (DominikGuzei) [inline packages/app/source/shared/startup.js:7]: Generic? :smile:

Comment (rhyslbw) [inline packages/app/source/shared/startup.js:7]: If someone wants to self-host Donations

Comment (DominikGuzei) [inline packages/app/source/shared/startup.js:7]: yep, we could (should) just move this into ENV vars …

**Pull request merged**